parquet-column/src/main/java/org/apache/parquet/column/impl/ColumnWriterV2.java

```diff
@@ -56,6 +57,7 @@
   private RunLengthBitPackingHybridEncoder definitionLevelColumn;
   private ValuesWriter dataColumn;
   private int valueCount;
+  private BloomFilterOpts opts;
```

Comment:
Could we name 'opts' to 'bloomFilterOpts' for increase readability along the code?